Create index.md (#2826)
github/explore: topics/browser-extension/index.md

---
display_name: Browser extension
short_description: A browser extension is a small software module for customizing a web browser.
topic: browser-extension
related: javascript, typescript
---

> A browser does typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.

Reviewer: How about this? 🤔 

```suggestion
Web browsers typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

(Apologies for the back-and-forth. Getting things worded clearly is tough!)

> A browser does typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.

Reviewer 2: The extensions themselves aren't cookie management, ad blocking, and the custom scripting and styling of webpages, those are functions performed by the extensions
```suggestion
Web browsers typically have extensions that can do a variety of things, including user interface modification, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

> A browser extension typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.

Reviewer 3: I do think that you should formulate it more like this:
`A browser does typically allow a ...`

Reviewer 4: No more $, #, no more stacking. Replace with your own. One that is unique to you and your device.. mine is similar to what follows but I also programmed each 'bot' to recognize my voice as the url, inside the url. You're the host, domain is the formula for your Unique.Realistic.Lobot..
~star2pltesq/%\pltesq2.2_[phrack{8.8}€]
